Source/missiles.cpp

```diff
@@ -1538,10 +1538,14 @@ void AddRuneExplosion(Missile &missile, const AddMissileParameter & /*parameter*
 		dmg = ScaleSpellEffect(dmg, missile._mispllvl);
 
 		missile._midam = dmg;
+		
+		constexpr Displacement Offsets[] = { Displacement(Direction::North), Displacement(Direction::NorthWest), { 0, 0 }, 
+											 Displacement(Direction::SouthEast), Displacement(Direction::West), Displacement(Direction::SouthWest), 
+											 Displacement(Direction::South), Displacement(Direction::East), Displacement(Direction::NorthEast)};
 
-		constexpr Displacement Offsets[] = { { -1, -1 }, { 0, -1 }, { 1, -1 }, { -1, 0 }, { 0, 0 }, { 1, 0 }, { -1, 1 }, { 0, 1 }, { 1, 1 } };
 		for (Displacement offset : Offsets)
 			CheckMissileCol(missile, dmg, dmg, false, missile.position.tile + offset, true);

```

Comment:
Since this hits all positions and the order isn't important it can be simplified using a range:
```suggestion
		auto searchArea = PointsInRectangleRangeColMajor { Rectangle { missile.position.tile, 1 } };
		for (Point position : searchArea)
			CheckMissileCol(missile, dmg, dmg, false, position, true);
```